keps/prod-readiness/sig-node/127.yaml

```diff
@@ -4,3 +4,5 @@
 kep-number: 127
 alpha:
   approver: "wojtek-t"
+beta:
+  approver: "wojtek-t"
```

Comment:
@wojtek-t can you please take a look? :)

Reply:
Will take a look later this week.
In the meantime, can you please work with SIG node on SIG approval?

Reply:
@wojtek-t Thanks! Yes, we are working with them since a few weeks already. I'll ping again :)